style.css

```diff
@@ -1,36 +1,40 @@
-#main{
-    background-color: #eee;
-    text-align: center;
-    padding: 2rem 1rem 0.5rem 1rem;
-    margin: 1.5rem 0;
+#main {
+  background-color: #eee;
+  text-align: center;
+  padding: 2rem 1rem 0.5rem 1rem;
+  margin: 1.5rem 0;
 }
 
-header{
-    margin: 2rem 1rem 1rem 1rem;
+header {
+  margin: 2rem 1rem 1rem 1rem;
 }
 
-#img-div{
-    background-color: white;
-    padding: 0.5rem 1rem;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
+#img-div {
+  background-color: white;
+  padding: 0.5rem 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ]  If you use flex boxes in the column direction you don't need to use justify-content: center to place them. You can delete those lines and note that they are not working already.

 

- [ ] There are some code repeats, try to use the DRY (Don't Repeat Yourself) method. You can create a property for them and add it to the your HTML:

    .flex {
    display: flex;
    }

and add it to the Div as class like
...
.

- [ ] One of the most important things. GitHub will be your CV someday. So, Having a good readme file important for you. Please fill that in as much as you can for now.

Reply:
Well done